packages/common/crypto/src/pkcs1/utils.spec.ts

```diff
@@ -0,0 +1,32 @@
+import BigNumber from 'bn.js';
+import { powerMod } from './utils';
+
+describe('powerMod function tests', () => {
+  test('powerMod with small numbers', () => {
```

Comment:
```suggestion
  it('powerMod with small numbers', () => {
```
you can refer https://github.com/toss/slash/pull/267#discussion_r1248051773

Reply:
Thank you for your advice. I modified the code as [follows]((https://github.com/toss/slash/pull/359/commits/56939f3cff36725eab0df1c9f77237e57ceedaec)).